src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -1615,6 +1615,20 @@ public function __call($method, $parameters)
             return $this->$method(...$parameters);
         }
 
+        if (Str::startsWith($method, 'get') || Str::startsWith($method, 'set')) {
+            $attribute = substr($method, 3);
+            $attribute = Str::snake($attribute);
```

Comment:
This will not work if I set `$snakeAttributes` on my model to `false`:

https://github.com/laravel/framework/blob/5.8/src/Illuminate/Database/Eloquent/Concerns/HasAttributes.php#L68-L73

Also, for example, if my table contains the `key` column, I will no longer able to get the model's primary key using:

```php
$model->getKey()
```